content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: "A Wonderful Hacktoberfest in Jenkins"
```

Comment:
I prefer titles that are short enough to fit on a single line when rendered.  The current title spans two lines and will make it a little more difficult for me to locate in the future because it does not include the year in the title.  Some of the past blog posts (2021, 2019, and 2018) included the year in the title.  Not consistent, but a preference for me.

This is an optional comment that you are welcome to ignore.
```suggestion
title: "Jenkins in Hacktoberfest 2022"
```

Reply:
I understand the comment @MarkEWaite and it makes sense. The purpose, I believe, was to have a catchy (journalistic) title. Something like "A great Hacktoberfest in Jenkins 2022 edition"
But this does not achieve the "snappy title" goal.